ale/drivers/__init__.py

```diff
@@ -36,6 +36,8 @@ def default(self, obj):
             return pvl.dumps(obj)
         if isinstance(obj, set):
             return list(obj)
+        if isinstance(obj, np.nan):
+            return None
```

Comment:
Previously there was a key that would resolve to nan if it was not needed. If the key is required and had nan, then you are correct that it would cause problems downstream, but we can't just remove them. It would to either error out entirely or warn and keep the NANs in regardless and punt the issue onto the user. Until we make a decision, this makes it so that you at least don't get silent errors when trying to unpack the JSON in the CSM plugin.